com.github.maoschanz.DynamicWallpaperEditor.json

```diff
@@ -0,0 +1,48 @@
+{
+    "app-id" : "com.github.maoschanz.DynamicWallpaperEditor",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "dynamic-wallpaper-editor",
+    "finish-args" : [
+        "--share=network",
```

Comment:
oops, not needed, i will correct that along with the tag/commit/version/release issue